components/requests/PaymentInformationCard.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box, Text, Divider, SimpleGrid, VStack } from '@chakra-ui/react'; // Chakra UI
+import PermitHolderInfoCard from '@components/internal/PermitHolderInfoCard'; // Custom Card component
+import { PaymentType } from '@lib/graphql/types'; // PayementType Enum
+import { MouseEventHandler } from 'react'; // React
+
+type PaymentInformationProps = {
```

Comment:
@OustanDing couldn't pass in an object as props here since we currently don't store these checkout details anywhere.